lib/koala/http_service.rb

```diff
@@ -1,5 +1,5 @@
 require 'faraday'
-require 'koala/http_service/multipart_request'
+require 'faraday/multipart' unless defined? Faraday::FilePart # hack for faraday < 1.9 to avoid warnings
```

Comment:
I had to add this little hack to avoid warning with faraday < 1.9

```
/home/ylecuyer/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/faraday-multipart-1.0.4/lib/faraday/multipart.rb:15: warning: already initialized constant Faraday::FilePart
/home/ylecuyer/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/faraday-1.7.0/lib/faraday/file_part.rb:53: warning: previous definition of FilePart was here
/home/ylecuyer/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/faraday-multipart-1.0.4/lib/faraday/multipart.rb:16: warning: already initialized constant Faraday::ParamPart
/home/ylecuyer/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/faraday-1.7.0/lib/faraday/param_part.rb:5: warning: previous definition of ParamPart was here
/home/ylecuyer/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/faraday-multipart-1.0.4/lib/faraday/multipart.rb:17: warning: already initialized constant Faraday::Parts
/home/ylecuyer/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/faraday-1.7.0/lib/faraday/file_part.rb:61: warning: previous definition of Parts was here
/home/ylecuyer/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/faraday-multipart-1.0.4/lib/faraday/multipart.rb:18: warning: already initialized constant Faraday::CompositeReadIO
/home/ylecuyer/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/faraday-1.7.0/lib/faraday/file_part.rb:66: warning: previous definition of CompositeReadIO was here
/home/ylecuyer/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/faraday-multipart-1.0.4/lib/faraday/multipart.rb:21: warning: already initialized constant Faraday::UploadIO
/home/ylecuyer/.rbenv/versions/2.7.6/lib/ruby/gems/2.7.0/gems/faraday-1.7.0/lib/faraday/file_part.rb:59: warning: previous definition of UploadIO was here
```